src/js/components/DateInput/DateInput.js

```diff
@@ -114,6 +115,7 @@ const DateInput = forwardRef(
         <Keyboard onEsc={open ? () => setOpen(false) : undefined}>
           <MaskedInput
             ref={ref}
+            id={id}
```

Comment:
Does it makes sense that both DropButton and the MaskedInput will be using the same `id`?

Reply:
Yes, we will never show both MaskedInput and DropButton at the same time.